pkg/yang/identity_test.go

```diff
@@ -388,19 +388,19 @@ var treeTestCases = []identityTestCase{
 				module: "base",
 				name:   "GREATGRANDFATHER",
 				values: []string{
-					"GRANDFATHER",
-					"GREATUNCLE",
+					"BROTHER", // Order is by hierarchy first then alphabetically
```

Comment:
This rule doesn't seem intuitive to me. If we're going to order the names I think it should be by the name only.

Reply:
now that the sort is done on the 2nd stage the order is alphabetical only